src/imports.rs

```diff
@@ -315,8 +322,16 @@ impl UseTree {
             visibility,
             attrs,
         };
+
+        let mut leading_modsep = false;
+
+        if context.config.edition() == Edition::Edition2018 {
+            if a.prefix.to_string().len() > 2 && &a.prefix.to_string()[0..2] == "::" {
```

Comment:
I think that we can use `a.prefix.starts_with("..")` instead of `&a.prefix.to_string()[0..2] == "::"`.